modules/agar/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Added
```

Comment:
Ahh... no changes were made to `agar`. I think you've put this in the wrong place, as this should be in `mcagar`.

- [ ] Fix changelogs and versions.txt to update `mcagar` instead of `agar`